spec/features/bulk_updates/bulk_updates_search_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+describe 'Searching collections to bulk update', reset_provider: true do
+  before(:all) do
+    # Create a few collections with unique attributes that we can search for
+    2.times { |i| publish_draft(version: "nasa.001#{i}") }
+    3.times { |i| publish_draft(short_name: "nasa.002#{i}") }
```

Comment:
I think setting up the test this way is fine, but an alternative could have been to switch providers to LARC or SEDAC and running a search against those collections, since they have been ingested.